src/Microsoft.NetCore.Analyzers/UnitTests/Performance/MutableStructsShouldNotBeUsedForReadonlyFieldsTests.Fixer.cs

```diff
@@ -0,0 +1,211 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.NetCore.CSharp.Analyzers.Performance;
+using Microsoft.NetCore.VisualBasic.Analyzers.Performance;
+using Test.Utilities;
+using Xunit;
+
+namespace Microsoft.NetCore.Analyzers.Performance.UnitTests
+{
+    public class MutableStructsShouldNotBeUsedForReadonlyFieldsFixerTests
+    {
+        [Fact]
+        public async Task CSharpReadonlyFieldsOfKnownMutableTypes_RemovesReadonlyModifier()
+        {
+            var initial = @"
+using System;
+using System.Threading;
+using System.Runtime.InteropServices;
+
+public class C
+{
+    private [|readonly|] SpinLock _sl = new SpinLock();
```

Comment:
Can you add C# and VB unit tests where you have a field declaration with multiple fields declared, such as `private readonly SpinLock _sl1 = new SpinLock(), sl2 = new SpinLock();`